src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,63 @@
 
```

Comment:
1. **Input validation**: While the code checks for division by zero in the `divide` method, it would be a good practice to also check for potential null inputs in any of the methods.
   
2. **Exception handling**: Instead of throwing a generic `IllegalArgumentException` for division by zero, consider throwing an `ArithmeticException`, as it is more specific to the issue.